pkg/apis/extensions/types.go

```diff
@@ -226,6 +226,14 @@ type DeploymentSpec struct {
 	// Value of this key is hash of DeploymentSpec.PodTemplateSpec.
 	// No label is added if this is set to empty string.
 	UniqueLabelKey string `json:"uniqueLabelKey,omitempty"`
+
+	// The config this deployment is rolling back to. Will be cleared after rollback is done.
+	Rollback *RollbackConfig `json:"rollback,omitempty"`
```

Comment:
Currently Rollback.Version (`int`) means the version to rollback to. `1` means roll back to version 1, `0` means it's unset and no-op. How do we specify that we want to roll back to the previous version without knowing the version number? How about `-1` means roll back to the latest previous version?